packages/next/build/webpack/config/blocks/css/loaders/file-resolve.ts

```diff
@@ -1,12 +1,8 @@
-export function cssFileResolve(
-  url: string,
-  _resourcePath: string,
-  urlImports: any
-) {
+export function cssFileResolve(url: string, _resourcePath: string) {
   if (url.startsWith('/')) {
     return false
   }
-  if (!urlImports && /^[a-z][a-z0-9+.-]*:/i.test(url)) {
+  if (/^[a-z][a-z0-9+.-]*:/i.test(url)) {
     return false
   }
```

Comment:
Returning false for a http url is incorrect when urlImports is enabled. It need to return true, since it should try to resolve the http url.